gocd/sp.target.gocd.yaml

```diff
@@ -63,6 +63,22 @@ pipelines:
                 sleep 600
             done
             osc -A https://api.suse.de/ api "/build/SUSE:SLE-15-SP5:GA:TEST/_result?view=summary&repository=images" | grep "result project" | grep 'code="published" state="published">' && echo PUBLISHED
+        - Release.Images.To.Test.New:
+          approval: manual
+          roles:
+          - SLE
+          environment_variables:
+            OSC_CONFIG: /home/go/config/oscrc-totest-manager
+            JIRA_PAT: /home/go/config/jira-pat
+            JIRA_INSTANCE: /home/go/config/jira-instance
+          resources:
+          - staging-bot
+          tasks:
+          - script: |-
+              sudo zypper ar -e https://download.opensuse.org/repositories/openSUSE:/Tools/openSUSE_Tumbleweed/openSUSE%3ATools.repo
```

Comment:
don't - add the tools to the container as needed.

Reply:
What container? I don't see the container that is being used in the `README.md` file of the gocd folder.

Reply:
Is it maybe this one? - https://build.opensuse.org/package/show/openSUSE:Tools:Images/gocd-agent-release-tools

Reply:
That's why I don't like writing documentation. You asked me for documentation, I provided you a link but you obviously didn't read it :smile: 

Yes, that's the container - gocd agents run in a fixed environment, they don't start fresh containers for every job as other CIs.

Reply:
The internal documentation you gave me privately does not contain the information about the container image I linked. I triple-checked everything before I asked. I am sorry if I overlooked this. I will open an SR that adds that package to the environment.

Reply:
Here the SR: https://build.opensuse.org/request/show/1062003

Reply:
The addition of the repo was removed. Thus resolving the conversation. Feel free to reopen if you have a different opinion.